ansible/roles/gpu/tasks/nvidia-gpu-RHEL.yaml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: RHEL subscription
+  redhat_subscription:
+    release: "{{ ansible_distribution_major_version }}.{{ ansible_distribution_minor_version }}"
+  when:
+    - not offline_mode_enabled
+
+- name:
+  rhsm_repository:
+    name: rhel-8-for-x86_64-baseos-eus-rpms
```

Comment:
assuming RHEL 9 will have similar name for its repos
`rhel-{{ ansible_distribution_major_version | int }}-for-x86_64-baseos-eus-rpms` should cover feature changes with
```
when:
  - "{{ ansible_distribution_major_version | int > 7 }}"
```